docs/perf-scale-benchmarks.md

```diff
@@ -59,9 +55,35 @@ taking specific assumptions into account. This section aims to provide clarity o
 
 ![vmiCreationToRunningSecondsP95 for VM](perf-scale-graphs/vm/vm-p95-Creation-to-Running.png "vmiCreationToRunningSecondsP95 for VM")
 
-#### Scalability benchmarks for v1.3.0 release
+#### CPU and Memory usage of virt-api and virt-controller 
 
-NOTE: The scalability metrics were hampered by a bug, that was fixed before the release [here](https://github.com/kubevirt/kubevirt/pull/12096). Hence the scalability metrics data needs to be re-evaluated by users with the bug fix to get a more accurate benchmarks
+Note: virt-api and virt-controller memory usage is being collected from June 27th
```

Comment:
Are the changes in data in CPU graphs related to the bugfix/

Reply:
I have added only the avg cpu and memory graphs in this doc, those are not related to bugfix.